src/main/java/makaroshyna/onlinebookstore/controller/AuthenticationController.java

```diff
@@ -24,8 +26,15 @@ public class AuthenticationController {
     @Operation(summary = "Register a new user",
             description = "Endpoint for registering new users")
     public UserRegistrationResponseDto register(
-            @RequestBody @Valid UserRegistrationRequestDto requestDto)
-            throws RegistrationException {
+            @RequestBody @Valid UserRegistrationRequestDto requestDto
+    ) throws RegistrationException {
         return userService.register(requestDto);
     }
+
+    @PostMapping("/login")
+    @Operation(summary = "Login a user",
+            description = "Endpoint to login users")
+    public UserLoginResponseDto login(UserLoginRequestDto requestDto) {
+        return userService.authorize(requestDto);
```

Comment:
```suggestion
        return userService.authorize(requestDto);
```
please use authenticationService and call the method `authenticate`